src/assets/locales/fr.json

```
@@ -90,25 +117,20 @@
   "close-room-modal.message": "En fermant cette salle, vous allez exclure tous les participants, vous y compris, et la fermer définitivement.{linebreak}Confirmer ? Cette action ne peut pas être annulée.",
   "close-room-modal.room-name-match-error": "Le nom de la salle ne correspond pas",
   "close-room-modal.title": "Fermer la salle",
-  "close-room-modal.type-to-confirm": "Entrer le nom de la salle pour confirmer: {roomName}",
+  "close-room-modal.type-to-confirm": "Entrer le nom de la salle pour confirmer : {roomName}",
   "content-menu.objects-menu-button": "Objets",
   "content-menu.people-menu-button": "Personnes ({presenceCount})",
```

```suggestion
  "content-menu.people-menu-button": "Personnes : ({presenceCount})",
```
If it's like a counter, better to put colon

---

It doesn't look good to me.

---

I have updated, I didn't see the parenthesis

---

I'm not for this change either. What do others think?